security/rs-security-best-practices.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Best Practices
+page_title: Best Security Practices when Installing and Using Telerik Report Server
+description: "Check the recommended security practices when installing the Telerik Report Server and working with its reports, users, and services."
+slug: rs-security-best-practices
+tags: telerik, reporting, report, server, security, best, practices
+published: True
+position: 4
+---
+
+# Security Best Practices
+
+The article discusses general and report server-specific security practices. While the recommendations may be necessary to keep your Telerik Report Server instance with all report definitions, data connections, etc. secure, they may not be sufficient. The article should not be regarded as a complete and comprehensive security guidance.
+
+## Configuration Settings
+
+The suggestions in this section are the responsibility of the administrator deploying the Telerik Report Server. They are general security settings of web applications deployed on the Windows IIS Servers. We have implemented most of them and let you switch them as configuration options when installing the Report Server:
+
+* Configure your Report Server instance to run under [HTTPS protocol](https://developer.mozilla.org/en-US/docs/Glossary/HTTPS). Consider the advice in the article [Configuring IIS Website to Work Over HTTPS]({%slug configuring-iis-website-to-work-over-https%}).
+* Let the user with lowered permissions in IIS and Report Server ServiceAgent be your preferred choice. By default, the MSI installer suggests applying the [principle of least privilege](https://learn.microsoft.com/en-us/entra/identity-platform/secure-least-privileged-access) and creating a dedicated Windows user named __ReportServerUser__ whose identity will be used by both applications. The user is granted the minimum necessary permissions to operate within the installation folder of Telerik Report Server as explained in [Report Server Installation]({%slug installation%}). 
+* Disable the [CORS]({%slug cors%}) if possible; or enable CORS only for trusted hosts.
+* Use the [Encryption]({%slug security%}#encryption) functionality of the Report Server to keep your assets safer.
+* Consider the [Rate Limiting]({%slug security%}#rate-limiter) in your Report Server to restrict the network traffic and prevent bad agents from exhausting system resources.
```

Comment:
bad agents -> 'malicious agents that can exhaust system resource by conducting Denial-of-Service attacks.'